pylint/checkers/classes.py

```diff
@@ -939,23 +939,22 @@ def _check_unused_private_functions(self, node: astroid.ClassDef) -> None:
                         break
             else:
                 name_stack = []
-                outer_level_names = ""
                 curr = parent_scope
                 # Generate proper names for nested functions
                 while curr != node:
                     name_stack.append(curr.name)
                     curr = curr.parent.scope()
 
-                if name_stack:
-                    outer_level_names = f"{'.'.join(reversed(name_stack))}."
+                outer_level_names = f"{'.'.join(reversed(name_stack))}"
 
-                function_repr = f"{outer_level_names}{function_def.name}({function_def.args.as_string()})"
```

Comment:
I really liked this line. It helped make the code more readable. In contrast to having a long f-string as an argument.